app/client/src/workers/common/AppComputationCache/index.ts

```
@@ -2,11 +2,17 @@ import { APP_MODE } from "entities/App";
 import localforage from "localforage";
 import isNull from "lodash/isNull";
 import loglevel from "loglevel";
-import { EComputationCacheName, type ICacheProps } from "./types";
+import {
+  EComputationCacheName,
+  type IValidatedCacheProps,
+  type ICacheProps,
+} from "./types";
 import debounce from "lodash/debounce";
+import { isFinite, isNumber, isString } from "lodash";
```

Biome (1.9.4)

[error] Do not shadow the global "isFinite" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)